src/converter/TypeScriptToKotlin.kt

```diff
@@ -180,7 +180,7 @@ class TypeScriptToKotlin(
         }
 
         val name = getName(rightNode)
-        qualifiedName += name
+        qualifiedName += name.split('/')
```

Comment:
Should ts2kt crash on provided test w/o this change? It doesn't for me.

Reply:
It crashes on similar case in react-router. Perhaps it can't be reproduced in test environment, I don't know.